tests/postgresql_test.py

```diff
@@ -1,9 +1,30 @@
+import re
+
+
 def test_postgresql_service(server):
     postgresql = server.service("postgresql")
     assert postgresql.is_running
     assert postgresql.is_enabled
 
-
 def test_postgresql_port(server):
     postgresql = server.addr("localhost")
     assert postgresql.port("5432").is_reachable
+
+
+def test_postgresql_databases(server):
+    result = server.run("podman exec postgresql psql -U postgres -c '\\l'")
+    assert "foreman" in result.stdout
+    assert "candlepin" in result.stdout
+    assert "pulp" in result.stdout
+
+
+def test_postgresql_users(server):
+    result = server.run("podman exec postgresql psql -U postgres -c '\\du'")
+    assert "foreman" in result.stdout
+    assert "candlepin" in result.stdout
+    assert "pulp" in result.stdout
+
+
+def test_postgresql_hba_config(server):
+    result = server.run("podman exec postgresql cat /var/lib/pgsql/data/pg_hba.conf")
+    assert re.search(r"host\s+all\s+all\s+127\.0\.0\.1/32\s+md5", result.stdout)
```

Comment:
You dropped the hba editing, so I think this test can be removed (and then the `re` import too)

Reply:
Done!